tee-worker/identity/client-sdk/packages/client-sdk/docs/modules/request.md

```diff
@@ -471,3 +472,43 @@ this is the signed payload.
 #### Defined in
 
 [lib/requests/transfer-native.request.ts:30](https://github.com/litentry/client-sdk/blob/develop/lib/requests/transfer-native.request.ts#L30)
+
+___
+
+### transferSolana
+
+▸ **transferSolana**(`api`, `data`): `Promise`\<\{ `payloadToSign?`: `string` ; `send`: (`args`: \{ `authentication`: `string`  }) => `Promise`\<\{ `blockHash`: `string` ; `extrinsicHash`: `string` ; `response`: `WorkerRpcReturnValue`  }\>  }\>
+
+Transfers Solana to another account on the Litentry Parachain.
```

Comment:
on Litentry Parachain -> on Solana network ? 😕

Reply:
It was a copy/paste of what we have for ethereum 😅 I'll update those comments